foundation/foundation-configuration-store/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  #%L
+  AIOps Foundation::AIOps MDA
+  %%
+  Copyright (C) 2021 Booz Allen
+  %%
+  This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+  #L%
+  -->
+
+<Configuration>
+	<Appenders>
+		<Console name="Console" target="SYSTEM_OUT">
+			<PatternLayout
+				pattern="%highlight{%d{HH:mm:ss.SSS} [%t] %-5level %logger{36}.%M():%L - %msg%n}{FATAL=red blink, ERROR=red, WARN=yellow bold, INFO=grey, DEBUG=green bold, TRACE=blue}" />
+		</Console>
+	</Appenders>
+	<Loggers>
+		<Logger name="com.boozallen.aiops" level="info" additivity="false ">
```

Comment:
A: The package root we're using here is `com.boozallen.aissemble` not aiops.  We also have an ongoing effort to move all current artifacts using the aiops package over to using aissemble instead.